<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardPilotVLMBoat</class>
 <widget class="QWidget" name="BoardPilotVLMBoat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>92</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QRadioButton" name="rd_heading">
     <property name="text">
      <string>Heading</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btn_setHeading">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: rgb(239, 243, 247);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="rd_angle">
     <property name="text">
      <string>Angle vent</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="tool_edtSpinBox" name="editAngle">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>190</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>190</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>94</green>
           <blue>168</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> Â°</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-180.000000000000000</double>
     </property>
     <property name="maximum">
      <double>180.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btn_angleFlip">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: rgb(239, 243, 247);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btn_setAngle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>18</width>
       <height>18</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>background-color: rgb(239, 243, 247);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QRadioButton" name="rd_ortho">
     <property name="text">
      <string>Ortho</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="rd_VMG">
     <property name="text">
      <string>VMG</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QRadioButton" name="rd_vbvmg">
     <property name="text">
      <string>VBVMG</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_Pilototo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pilototo (x/x)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_clearPilototo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Effacer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="tool_edtSpinBox" name="editHeading">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>190</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>11</red>
           <green>190</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>94</green>
           <blue>168</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="HighlightedText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>85</red>
           <green>85</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string> Â°</string>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>359.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tool_edtSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>BoardTools.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rd_heading</sender>
   <signal>toggled(bool)</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_rdToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>31</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rd_angle</sender>
   <signal>toggled(bool)</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_rdToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rd_VMG</sender>
   <signal>toggled(bool)</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_rdToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rd_ortho</sender>
   <signal>toggled(bool)</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_rdToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rd_vbvmg</sender>
   <signal>toggled(bool)</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_rdToggle(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editHeading</sender>
   <signal>valueChanged(double)</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_headingValueChg(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editAngle</sender>
   <signal>valueChanged(double)</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_angleValueChg(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_angleFlip</sender>
   <signal>clicked()</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_flipAngle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_setHeading</sender>
   <signal>clicked()</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_setHeading()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_setAngle</sender>
   <signal>clicked()</signal>
   <receiver>BoardPilotVLMBoat</receiver>
   <slot>slot_setAngle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_rdToggle(bool)</slot>
  <slot>slot_headingValueChg(double)</slot>
  <slot>slot_angleValueChg(double)</slot>
  <slot>slot_flipAngle()</slot>
  <slot>slot_setHeading()</slot>
  <slot>slot_setAngle()</slot>
  <slot>slot_pilototo()</slot>
  <slot>slot_clearPilototo()</slot>
 </slots>
</ui>
